.github/workflows/rust-ci.yaml

```diff
@@ -23,10 +23,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      # When rustup is updated, it tries to replace its binary, which on Windows is somehow locked.
```

Comment:
idk why this was necessary, but if it was necessary before, adding mold wouldn't make it unnecessary

Reply:
The comment was there to explain why they were disabling the auto-self-update before invoking the toolchain setup. Since we're no manually using rustup to change the toolchain, we no longer need to worry about disabling the self-updating.